src/rpc/ngHandlers/NFTInfo.cpp

```diff
@@ -66,7 +63,12 @@ void
 tag_invoke(boost::json::value_from_tag, boost::json::value& jv, NFTInfoHandler::Output const& output)
 {
     // TODO: use JStrings when they become available
-    auto object = boost::json::object{
+    // TODO - this formatting is exactly the same and SHOULD REMAIN THE SAME
+    // for each element of the `nfts_by_issuer` API. We should factor this out
+    // so that the formats don't diverge. In the mean time, do not make any
+    // changes to this formatting without making the same changes to that
+    // formatting.
+    auto const object = boost::json::object{
```

Comment:
Now that there is no "injection" there is no need to jump thru hoops with `std::move` and temporary object. Safe to rewrite to something like
```cpp
jv = { ... };
```

Btw., I appreciate that you try to add `const` everywhere but in this particular case making `object` const actually prevents `std::move` below from doing its job and makes a copy instead :)